docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,459 @@
+  Gracefully stopping... (press Ctrl+C again to force)
+  Stopping compose_tp_xo_python_1 ... done
+  Stopping compose_tp_config_1 ... done
+  Stopping compose_client_1 ... done
+  Stopping compose_rest_api_1 ... done
+  Stopping compose_tp_intkey_python_1 ... done
+  Stopping compose_validator_1 ... done
+
+  % docker-compose -f sawtooth-default.yaml down
+
+
+
+.. note::
+
+  Genesis block, validator and transaction processor startup are handled for you
+  if you're using the Docker workflow. If you want a better understanding of
+  this process, see :doc:`ubuntu`.
+
+
+Confirming Connectivity 
+=======================
+
+To confirm that a validator is up and running, and reachable from the client container, 
+you can use the curl command. 
+
+First, install curl on the client container, by running the following commands 
+from the client container terminal window:
+
+.. code-block:: console
+
+	$ sudo sed -i -e 's/us.archive.ubuntu.com/archive.ubuntu.com/g' /etc/apt/sources.list
+	$ apt-get update
+	$ apt-get install curl
+
+You can now use the curl command to test the connectivity betweent the client container
+and the validator:
+
+.. code-block:: console
+
+	root@75b380886502:/# curl http://rest_api:8080/blocks
+
+If the validator is running and reachable, the output should be similar to:
+
+.. code-block:: console
+
+	{
+	  "data": [
+	    {
+	      "batches": [],
+	      "header": {
+	        "batch_ids": [],
+	        "block_num": 0,
+	        "consensus": "R2VuZXNpcw==",
+	        "previous_block_id": "0000000000000000",
+	        "signer_pubkey": "03061436bef428626d11c17782f9e9bd8bea55ce767eb7349f633d4bfea4dd4ae9",
+	        "state_root_hash": "708ca7fbb701799bb387f2e50deaca402e8502abe229f705693d2d4f350e1ad6"
+	      },
+	      "header_signature": "119f076815af8b2c024b59998e2fab29b6ae6edf3e28b19de91302bd13662e6e43784263626b72b1c1ac120a491142ca25393d55ac7b9f3c3bf15d1fdeefeb3b"
+	    }
+	  ],
+	  "head": "119f076815af8b2c024b59998e2fab29b6ae6edf3e28b19de91302bd13662e6e43784263626b72b1c1ac120a491142ca25393d55ac7b9f3c3bf15d1fdeefeb3b",
+	  "link": "http://rest_api:8080/blocks?head=119f076815af8b2c024b59998e2fab29b6ae6edf3e28b19de91302bd13662e6e43784263626b72b1c1ac120a491142ca25393d55ac7b9f3c3bf15d1fdeefeb3b",
+	  "paging": {
+	    "start_index": 0,
+	    "total_count": 1
+	  }
+	}root@75b380886502:/#
+
+If the validator process, or the validator container, is not running, the curl
+command will return nothing, or time out.
+
+Finding Log Files
+=================
+
+.. todo:: 
+	
+	Info on finding log files within docker environment
+
+Transaction Processors
+======================
+
+The Docker Compose file (*sawtooth-default.yaml*) starts the following
+transaction processors automatically:
+
+* tp_config
+* tp_intkey_python
+* tp_xo_python
+
+These transaction processors handle transactions for the config, intkey, and XO transaction families, and are written in Python. 
+
+Sawtooth includes additional transaction processors.
```

Comment:
Let's not maintain this list here.  There are not docker containers published for all of these yet.